### ISSUE TITLE
Database would not start using docker-compose and Digital Ocean

### ISSUE THREAD

**Issue by stanleyrya** [molecular-playground/databaes#4]
I apologize for not putting too many details, however this is how you can reproduce the error:

```
docker-compose stop
docker-compose rm
docker volume rm molecularplayground_postgres-data
docker-compose up
```

As a "hotfix" to get around the error while deploying follow the following instructions:
1. Remove the `molecularplayground_postgres-data` volume
2. Create a postgres container as described [here](https://github.com/molecular-playground/databaes) using `molecularplayground_postgres-data` as the volume name
3. `docker stop postgres && docker rm postgres`
4. `docker-compose build && docker-compose up`

**Comment by stanleyrya** [comment]
Here's what the issue spits out:
`FATAL: could not map anonymous shared memory: Cannot allocate memory`

<img width="995" alt="screen shot 2016-04-15 at 4 53 06 pm" src="https://cloud.githubusercontent.com/assets/8700100/14574677/07ea32d4-032b-11e6-8fcb-f5a6180e4721.png">

**Comment by stanleyrya** [comment]
Turns out we just did not have enough memory to run docker-compose up. Ideally the other microservices should wait until postgres has completed initializing because they depend on it, however I guess postgres makes it seam like it has completed initializing before it actually has finished initializing. Anyway, I detailed some instructions to help work around this issue in [the main repo](https://github.com/molecular-playground/molecular-playground/commit/1378ca589ecc1946713a3d6a396cc8dd05334e07).
